### PR TITLE
track the charge id in ga

### PR DESCRIPTION
**Pull request by AWare**
Add the stripe charge id in google analytics.
@ajosephides @rtyley @paulbrown1982

### CODE REVIEW

File: app/controllers/Giraffe.scala [hunk 1]
```diff
@@ -124,19 +124,18 @@ class Giraffe(paymentServices: PaymentServices) extends Controller {
   }
 
   def thanks(countryGroup: CountryGroup) = NoCacheAction { implicit request =>
-
+    val charge = request.session.get(chargeId)
```
Comment (paulbrown1982) [inline app/controllers/Giraffe.scala:127]: Where's the chargeId actually added to the session?

Comment (AWare) [inline app/controllers/Giraffe.scala:127]: https://github.com/guardian/contributions-frontend/blob/85cc65f5c41644d23682df9cf00d5c58fc2f7b9f/app/controllers/Giraffe.scala#L172-L172

File: assets/javascripts/src/modules/analytics/ga.es6 [hunk 2]
```diff
@@ -54,14 +59,24 @@ export function init() {
     ga('set', dimensions.signedIn, isLoggedIn.toString());
     ga('set', dimensions.isLoggedOn, isLoggedIn.toString());
     ga('set', dimensions.signedOut, signedOut.toString());
-    ga('set', dimensions.platform, 'contributions');
+    ga('set', dimensions.platform, 'membership');
```
Comment (alexduf) [inline assets/javascripts/src/modules/analytics/ga.es6]: why is that changing from contributions to membership?

File: assets/javascripts/src/modules/analytics/ga.es6 [hunk 3]
```diff
@@ -54,14 +59,24 @@ export function init() {
...
     if (isLoggedIn) {
         ga('set', dimensions.identityId, u.id);
     }
     if (guardian.ophan) {
         ga('set', dimensions.ophanPageViewId, guardian.ophan.pageViewId);
     }
+    console.log(guardian);
+    debugger;
```
Comment (alexduf) [inline assets/javascripts/src/modules/analytics/ga.es6]: are these two lines supposed to go to PROD?

**Comment by rtyley** [comment]
Could we record in this PR why we're adding the stripe charge id to Google Analytics?

**Comment by AWare** [comment]
^ @ajosephides

File: assets/javascripts/src/modules/analytics/ga.es6 (hunk 3 above)
Comment (AWare) [inline assets/javascripts/src/modules/analytics/ga.es6]: Nope. Ooops.

**Comment by alexduf** [comment]
I have no other comments :+1:

**Comment by paulbrown1982** [comment]
👍

**Pull request merged**